exchange/bitswap/workers.go

```diff
@@ -9,16 +11,31 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 16
+
+func init() {
+	twc := os.Getenv("IPFS_TASK_WORKERS")
+	if twc != "" {
+		n, err := strconv.Atoi(twc)
+		if err != nil {
+			return
```

Comment:
a panic would be less damaging. `os.Exit(...)` can put things into a really bad state if called at the wrong time. I think printing an error and continuing with the default value will be the best option